Kernel/Arch/i386/CPU.cpp

```diff
@@ -788,10 +788,20 @@ void Processor::cpu_detect()
         set_feature(CPUFeature::TSC);
     if (processor_info.ecx() & (1 << 30))
         set_feature(CPUFeature::RDRAND);
-
+    if (processor_info.edx() & (1 << 11)) {
+        u32 stepping = processor_info.eax() & 0xf;
+        u32 model = (processor_info.eax() >> 4) & 0xf;
+        u32 family = (processor_info.eax() >> 8) & 0xf;
```

Comment:
The more of these flags we check, the more it feels like a bunch of defines or enums for the feature flags might be useful...